src/assignment/views/AssignmentEdit.tsx

```diff
@@ -19,7 +19,7 @@ import {
 	StickyEdgeBar,
 	Tabs,
 } from '@viaa/avo2-components';
-import { AssignmentBlock } from '@viaa/avo2-types/types/assignment';
+import { AssignmentBlock, AssignmentSchema_v2 } from '@viaa/avo2-types/types/assignment';
```

Comment:
de schema suffix blijft bij voorkeur in de typings repo, en wordt dan daar ge-exporteerd als: Assignment_v2
Mogelijks kan je die reeds gebruiken via: `Avo.Assignment.Assignment_V2`